### PR TITLE
fileutil, src: format errors

### PR DESCRIPTION
**Pull request by ZYunH**
Format errors, according to https://github.com/golang/go/wiki/CodeReviewComments#error-strings

### CODE REVIEW

File: pkg/fileutil/lock_windows.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ var (
 	modkernel32    = syscall.NewLazyDLL("kernel32.dll")
 	procLockFileEx = modkernel32.NewProc("LockFileEx")
 
-	errLocked = errors.New("The process cannot access the file because another process has locked a portion of the file.")
+	errLocked = errors.New("the process cannot access the file because another process has locked a portion of the file")
```
[the remark this answers is not in the document]
Comment (ZYunH) [reply pkg/fileutil/lock_windows.go]: Sure, is it correct now?

[the remark this answers is not in the document]
Comment (spzala) [reply pkg/fileutil/lock_windows.go]: Well no :-) The **commit title** needs to be changed (not the PR title) - something like `pkg: format errors` Thanks!

[the remark this answers is not in the document]
Comment (ZYunH) [reply pkg/fileutil/lock_windows.go]: oops, my bad :0
Now the fmt test is passed. But some of the tests have failed, looks like it is unrelated to my pull request?

[the remark this answers is not in the document]
Comment (spzala) [reply pkg/fileutil/lock_windows.go]: No problem. Thanks and as you noticed test failures is unrelated. Thanks @jingyih for the review!

**Pull request merged**